src/client/dfuse/dfuse_main.c

```diff
@@ -195,6 +200,16 @@ main(int argc, char **argv)
 		D_GOTO(out_dfuse, ret = -DER_INVAL);
 	}
 
+	if (!dfuse_info->di_foreground) {
+		rc = daemon(0, 0);
```

Comment:
why not use fuse_daemonize?

Reply:
Good question, I wasn't aware of this function but I can't see anything it offers over what I daemon() does?

Reply:
i don't know either.. it just seems if fuse offers this functionality maybe better to use it?
i haven't looked into the implementation of fuse_daemonize if it adds something extra.